### PR TITLE
Tested PR for MRIO, including image data set and child MRI terms, per 1481

### PR DESCRIPTION
**Pull request by adbartni**
Added image data set and subclasses from MRIO, edited imaging assay and magnetic resonance imaging assay, and added image data set analysis class.

### CODE REVIEW

**Comment by jamesaoverton** [comment]
Sorry, I was not clear: This PR needed to be reviewed and approved by the OBI developers before it was merged. I'm going to revert the commit.

**Comment by jamesaoverton** [comment]
@adbartni You followed the https://github.com/obi-ontology/obi#making-changes section of the README, which is much appreciated. That documentation does say "When all tests have passed, your PR can be merged into master", which is what you did. But we do need a chance to look at the details of your changes (in OWL) before they are merged, especially since this is your first OBI PR.

Please open a new PR with these changes and we can discuss it. I will be the one to merge the changes.

File: src/ontology/obi-edit.owl [hunk 1]
```diff
@@ -20912,6 +20912,41 @@ b) that the amount of analyte/measurand detected is over some predetermined thre
     
 
 
+    <!-- http://purl.obolibrary.org/obo/OBI_0002985 -->
+
+    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/OBI_0002985">
```
Comment (jamesaoverton) [inline src/ontology/obi-edit.owl:20917]: @beckyjackson This assay term snuck into `obi-edit.owl`, and the SPARQL "branch" tests didn't catch it. Can we update the tests to catch this case?

File: src/ontology/obi-edit.owl [hunk 2]
```diff
@@ -21882,6 +21917,645 @@ b) that the amount of analyte/measurand detected is over some predetermined thre
...
+
+    <!-- http://purl.obolibrary.org/obo/OBI_0003337 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/OBI_0003337">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0003336"/>
+        <obo:IAO_0000115>A type of functional MRI in which the participant is not given a task, but just rests during acquisision</obo:IAO_0000115>
+        <obo:IAO_0000117>https://orcid.org/0000-0001-9676-7377 &quot;Alexander D. Bartnik&quot;</obo:IAO_0000117>
+        <obo:IAO_0000117>https://orcid.org/0000-0001-9990-8331 &quot;Alexander D. Diehl&quot;</obo:IAO_0000117>
+        <obo:IAO_0000117>https://orcid.org/0000-0002-2104-0568 &quot;Lucas M. Serra&quot;</obo:IAO_0000117>
+        <rdfs:label xml:lang="en">Resting state functional MR image data set</rdfs:label>
```
Comment (jamesaoverton) [inline src/ontology/obi-edit.owl:22153]: OBI uses lower case labels, so "Resting" should be "resting". We also avoid acronyms in primary labels, so I think that "MR" should be expanded. Sometimes exceptions are made for clarity and length.

File: src/ontology/obi-edit.owl [hunk 3]
```diff
@@ -21882,6 +21917,645 @@ b) that the amount of analyte/measurand detected is over some predetermined thre
     
 
 
+    <!-- http://purl.obolibrary.org/obo/OBI_0003327 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/OBI_0003327">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/IAO_0000100"/>
+        <obo:IAO_0000115>A data set that is comprised of structured measurements about some entity and its associated metadata using pixels (2D), voxels (3D), or an arbitrary number of dimensions. An image data set can be the source from which an image is produced.</obo:IAO_0000115>
+        <obo:IAO_0000117>https://orcid.org/0000-0001-9625-1899 &quot;William D. Duncan&quot;</obo:IAO_0000117>
```
Comment (jamesaoverton) [inline src/ontology/obi-edit.owl:21925]: These `&quot;`s can be problematic during file conversion using various tools. Can they be omitted?

**Comment by jamesaoverton** [comment]
One last general comment: OBI developers have been pushing to use templates more and more. Could some of these terms be put in a new template? The image and image dataset types seem like good candidates.

**Comment by adbartni** [comment]
@jamesaoverton Sorry, I mistook the automatic test to be a "go" for merging. I can certainly create a template for the image data set classes and clean up the formatting re: the capitals and quotes.

**Comment by jamesaoverton** [comment]
Thanks, and no need to apologize. You followed the documentation, which is much appreciated, but the documentation was incorrect. I've updated it.

**Comment by adbartni** [comment]
@jamesaoverton I'm a little confused about which template I should be using for `image data set` and `image data set analysis`. I don't see an existing template that holds `data sets` or `data transformations`, is there one that I should use? I see `imaging assay` is in the assays template, so would it make more sense to edit `imaging assay` to output `image data set` in the template and add the rest of the terms to the OWL file?

**Comment by jamesaoverton** [comment]
This PR includes a bunch of dataset terms, so I'm suggesting that you create a table of the data set terms you want to add, which we can turn into a new template.

How many data transformations are you adding? If there's just a handful, we shouldn't bother with a template.

**Comment by adbartni** [comment]
Gotcha. As of now it's only two data transformations in this PR, and ~18 planned for a future PR. Should I create a template for just image data set, or a larger template for data set in general?

**Comment by adbartni** [comment]
I found creating two new templates with dependencies between them to be difficult when it came to testing, so I just made a new one for data sets and added the two data transformation classes directly to obi-edit.owl. When I do the PR for the MRI analyses terms I'll create a data transformations template and add them there, since the dependencies should already be in OBI at that point. I can close this out and remove the branch if everything looks good.

**Pull request merged**